hack/update-rhcos-bootimage.py

```diff
@@ -23,6 +23,12 @@
     string_f = codecs.getreader('utf-8')(f)  # support for Python < 3.6
     meta = json.load(string_f)
 newmeta = {}
+
+# x86_64 boot image bumps should contain more than just the ostree tarball and
+# qemu qcow2 image
+if args.arch == 'amd64' and len(meta['images']) <= 2:
```

Comment:
`2` seems pretty brittle.  Can we spot-check for specific artifacts we expect to exist?

Reply:
We can...though that means maintaining an allowlist of artifacts we expect, which has it's own challenges.

I'll take another swing at it with the list of artifacts.

Reply:
I'm not proposing listing all the affected artifacts, just checking for a couple well-known ones like `aws`.  The problem with `2` is that if light builds start having 3 artifacts, they'll pass this test.